proposals/coredns.adoc

```diff
@@ -0,0 +1,103 @@
+== CoreDNS Proposal
+
+*Name of project:* CoreDNS
+
+*Description*
+
+CoreDNS is a fast, flexible and modern DNS server. Its performant and flexible implementation allows CoreDNS to be easily extended to support various data sources and to implement rich DNS service behaviors: for example, response caching, query rewrite, load-balancing, zone transfer and signing.
+CoreDNS is the successor of SkyDNS (https://github.com/skynetservices/skydns), a DNS server that uses etcd as its datastore backend. SkyDNS is widely
+used in cloud deployments, but lacks the flexibility of CoreDNS.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle
+
+*Preferred Maturity Level*: Inception
+
+*Unique Identifier*: coredns
+
+*License*: Apache License v2.0
+
+*Source control repositories*: https://github.com/miekg/coredns
+
+*Initial Committers*:
+
+* Miek Gieben github: miekg
+* Michael Richmond github: mrichmon
+* github: splack
+* Felix Cantournet github: fcantournet
+* github: leelynne
+* Matt Layher github: mdlayher
+* Vasily Vailyev github: pixelbender
+* John Belamaric github: johnbelamaric
+* Chris O'Haver github: chrisohaver
+* Yong Tang github: yongtang
+
+*Infrastructure requirements (CI / CNCF Cluster)*: N/A
+
+*Issue tracker*: https://github.com/miekg/coredns
+
+*Website*: https://coredns.io
+
+*Release methodology and mechanics*: The current version is v004; there have been four official releases. Releases are not on a specific schedule
+but are coordinated by the community via the issue tracker. Additionally, the rule is that the master branch is production-ready at all times.
+A more formal release process is on its way, and may introduce semantic versioning, but a final decision has not yet been made. Precompiled binaries are
+distributed via GitHub and Caddy's download website (https://caddyserver.com/download), where "DNS" will be a Server Type option.
+
+*Social media accounts*: Twitter: @corednsio
+
+*Existing sponsorship*: Infoblox contributing developer time to implement CoreDNS->Kubernetes integration component and other enhancements.
+
+*Existing community*: The community is small, but growing. On GitHub, we have 50+ forks, 550+ stars, and 24 contributors (5 with merge privileges).
+Current number of Twitter followers is 180+. By aligning ourselves with the Caddy community, we hope to leverage Caddy's
+popularity for CoreDNS. By positioning CoreDNS as a better SkyDNS, we hope to entice existing users of SkyDNS to migrate to and embrace CoreDNS.
```

Comment:
Might be worth calling out if there is a migration path/guidance for existing SkyDNS users